engine/src/main/java/org/terasology/rendering/world/RenderableWorldImpl.java

```diff
@@ -101,8 +102,13 @@
     @Override
     public void onChunkLoaded(Vector3i chunkCoordinates) {
         if (renderableRegion.encompasses(chunkCoordinates)) {
-            chunksInProximityOfCamera.add(chunkProvider.getChunk(chunkCoordinates));
-            Collections.sort(chunksInProximityOfCamera, new ChunkFrontToBackComparator());
+            Chunk chunk = chunkProvider.getChunk(chunkCoordinates);
+            if (chunk == null) {
```

Comment:
Is this the most typical case? I would have thought most chunks aren't null and such I'd suggest flipping the if/else block.

Reply:
This is the exception, not the norm. I'll flip it right away.